test/src/main/java/org/apache/accumulo/test/fate/FateITBase.java

```diff
@@ -591,7 +591,13 @@ protected void testShutdownDoesNotFailTx(FateStore<TestEnv> store, ServerContext
     fate.seedTransaction(TEST_FATE_OP, txid, new TestRepo("testShutdownDoesNotFailTx"), true,
         "Test Op");
 
-    assertEquals(TStatus.SUBMITTED, getTxStatus(sctx, txid));
+    // The Fate operation could be in a SUBMITTED state if the
+    // Fate transaction runner thread has not picked it up yet.
+    // If the thread has picked it up, then it will be in an
+    // IN_PROGRESS state, but will be waiting on the finishCall
+    // latch to be called to continue.
+    assertTrue(TStatus.SUBMITTED == getTxStatus(sctx, txid)
+        || TStatus.IN_PROGRESS == getTxStatus(sctx, txid));
```

Comment:
Could we do a `Wait.for(()->TStatus.IN_PROGRESS == getTxStatus(sctx, txid))` or is it not guaranteed to reach IN_PROGRESS?

Also were you seeing test failures that prompted this change?

Reply:
This was based on a discussion with @kevinrr888. I think at this place it's guaranteed to be one or the other, I'm not sure we need a wait. @kevinrr888 ?